src/main/java/com/aiykr/iquais/IquaisApplication.java

```diff
@@ -4,11 +4,13 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
 
 /**
  * Main application class for the Iquais application.
  */
 @SpringBootApplication
+@ComponentScan
```

Comment:
why is this required, doesn't @SpringBootApplication already have it?

Reply:
yes. i was checking for swagger2 enable